akka-bbb-apps/src/main/scala/org/bigbluebutton/core/BigBlueButtonActor.scala

```diff
@@ -102,6 +102,8 @@ class BigBlueButtonActor(
     } yield {
       log.debug("FORWARDING Register user message")
       m.actorRef forward (msg)
+      val eventWaitingTime = MsgBuilder.buildNotifyWaitingTimeReqMsg(msg.header.meetingId, msg.body.intUserId)
+      m.actorRef forward (eventWaitingTime)

```

Comment:
I'm missing the point of this request over here.

When akka-apps receives a register user message from bbb-web it requests itself to notify bbb-web back about the time the user registered? Why this timestamp can't be generated directly at bbb-web at the user registration?